gtk-sys/src/lib.rs

```diff
@@ -11766,9 +11766,9 @@ extern "C" {
     pub fn gtk_tree_sortable_get_type() -> GType;
     pub fn gtk_tree_sortable_get_sort_column_id(sortable: *mut GtkTreeSortable, sort_column_id: *mut c_int, order: *mut GtkSortType) -> gboolean;
     pub fn gtk_tree_sortable_has_default_sort_func(sortable: *mut GtkTreeSortable) -> gboolean;
-    pub fn gtk_tree_sortable_set_default_sort_func(sortable: *mut GtkTreeSortable, sort_func: GtkTreeIterCompareFunc, user_data: gpointer, destroy: glib::GDestroyNotify);
+    pub fn gtk_tree_sortable_set_default_sort_func(sortable: *mut GtkTreeSortable, sort_func: *mut c_void, user_data: gpointer, destroy: *mut c_void/*glib::GDestroyNotify*/);
```

Comment:
Can't do that, sorry. ;)
With the temporary `cast_*` exception, editing the `sys` files doesn't make sense. If a mistake is found there, we should fix the generator instead. But it doesn't seem like there's a mistake here.